### DISCUSSION TITLE
Camera stares into the abyss (probably the floor actually) when watching a replay.

### DISCUSSION THREAD

**Discussion by YaBoiBanic** [kinsi55/CS_BeatSaber_Camera2#71] [Q&A] [answered]
I've been trying to get my first person camera to be used in replays and nothing has worked. I only have one camera as I assumed all the others I was given would mess with this. I still can't get it to work! Any help?

`{ "visibleObjects": { "Walls": "Visible", "Debris": true, "UI": true, "Avatar": "Visible", "Floor": true, "CutParticles": true, "Notes": "Visible", "EverythingElse": true, "Sabers": true }, "type": "FirstPerson", "FOV": 90.0, "layer": 2, "antiAliasing": 1, "renderScale": 1.3, "orthographic": false, "farZ": 5000.0, "viewRect": { "x": 0.0, "y": 0.0, "width": 1.0, "height": 1.0, "locked": false }, "Multiplayer": { "followSpectatorPlattform": true }, "Smoothfollow": { "position": 10.0, "rotation": 5.0, "forceUpright": true, "followReplayPosition": false, "pivotingOffset": true }, "ModmapExtensions": { "autoOpaqueWalls": false, "autoHideHUD": false }, "FPSLimiter": { "fpsLimit": 120 }, "PostProcessing": { "transparencyThreshold": 0.0, "forceDepthTexture": false }, "targetPos": { "x": 0.0, "y": 0.0, "z": 0.0 }, "targetRot": { "x": 0.0, "y": 0.0, "z": 0.0 } }`

**Comment by kinsi55** [comment, marked as answer]
Well

> "followReplayPosition": false
